### ISSUE TITLE
declaration-block-single-line-max-declarations

### ISSUE THREAD

**Issue by magsout** [stylelint/stylelint#914]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

it's a bug

> Which rule, if any, is this issue related to?

declaration-block-single-line-max-declarations

> What CSS is needed to reproduce this issue?

```css
  .r-Grid--withGutter { margin: 0 calc(-0.5 * var(--r-Grid-gutter)) }

```

> What stylelint configuration is needed to reproduce this issue?

```json
  "stylelint": {
    "extends": "stylelint-config-standard",
    "rules": {
      "indentation": [ 2, {
      "hierarchicalSelectors": true
      }]
    }
  },
```

> Which version of stylelint are you using?

```json
  "devDependencies": {
    "stylelint": "^5.0.1",
    "stylelint-config-standard": "^4.0.1"
  },
```

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`postcss -c package.json -u postcss-import -u postcss-cssnext -u stylelint -u postcss-reporter -d ./build *.css`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

```shell
11:23	⚠  Expected a maximum of 1 declaration(s) (declaration-block-single-line-max-declarations) [stylelint]

```

**Comment by jeddy3** [comment]
In the [Usage examples](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/postcss-plugin.md#usage-examples) section of the [PostCSS plugin guide](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/postcss-plugin.md) we recommend either creating a separate lint task, or putting stylelint as the first plugin pipeline.

As your already using the CLI, then perhaps using the `stylelint` CLI is best for you:

```shell
stylelint path-to-your-css/*.css
```

Otherwise, try putting stylelint as the first plugin pipeline:

```shell
`postcss -c package.json -u stylelint -u postcss-import -u postcss-cssnext  -u postcss-reporter -d ./build *.css
```

Right now you're linting the output of `postcss-import` and `postcss-cssnext`. I suspect `postcss-cssnext` is adding a second declaration to your rule (perhaps if `--r-Grid-gutter` is a `rem` value?)

**Comment by jeddy3** [comment]
Closing.

@magsout Feel free to reopen the issue if I haven't diagnosed it correctly.

**Comment by magsout** [comment]
@jeddy3 yes, you're of course right ;) I used `-u postcss-import`  because of my `@import` directive that `stylelint`  didn't resolve, so I have to lint every single file.

**Comment by jeddy3** [comment]
I'm not sure what configuration options `postcss-cli` accepts, but perhaps using `postcss-imports`'s `plugins` option will work for you. As shown in [this example](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/postcss-plugin.md#example-b)
